src/IPNFT.sol

```diff
@@ -134,15 +138,29 @@ contract IPNFT is
         return reservationId;
     }
 
+    /**
+     * @notice deprecated: the old interface without a symbol.
+     */
+    function mintReservation(address to, uint256 reservationId, uint256 mintPassId, string memory tokenURI)
```

Comment:
Are we going to keep this func? As far as I see we are sending empty string on FE if the user doesn't give an input, so seems like this first mintReservation imp. will be obsolete :)

Reply:
but it's still usable. Imo a symbol shouldn't be *mandatory* to mint an IPNFT.